modules/plugin/jdbc/jdbc-sqlserver/README.txt

```diff
@@ -4,33 +4,18 @@ GeoTools SQL Server DataStore README
 JDBC Driver
 -----------
 
-To use the sql server datastore the jdbc 3.0 driver must be downloaded from 
-Microsoft:
+The current tested driver version is 7.2.1 for SQL Server and SQL Azure. According to Microsoft this
+driver provides access to SQL Azure, SQL Server 2017 from any Java application, application server, 
+or Java-enabled applet.
 
-  http://www.microsoft.com/download/en/details.aspx?displaylang=en&id=19847
-
-The current tested driver version is 3.0 for SQL Server and SQL Azure. According to Microsoft this
-driver provides access to SQL Azure, SQL Server 2008 R2, SQL Server 2008, SQL Server 2005 and 
-SQL Server 2000 from any Java application, application server, or Java-enabled applet.
-
-Make sure to unpack the sqljdbc4.jar file, as that's the one providing support for the Java
-version in this series.
+This plugin will automatically download the MS SQL Server JDBC driver from the central maven repository.
```

Comment:
So wait, now everything is on the mave repos?

Reply:
Well duh, looks like it is: https://mvnrepository.com/artifact/com.microsoft.sqlserver/mssql-jdbc

Reply:
Now if only oracle could play nice :P